src/api/app/views/webui/request/beta_show_tabs/_conversation.html.haml

```diff
@@ -7,24 +7,19 @@
         %i No description set
 .row
   .col-md-4.order-md-2.order-sm-1.mb-4#side-links
-    - if accepted_reviews.present? || declined_reviews.present? || open_reviews.present?
+    - if request_reviews.present?
       .d-flex.justify-content-between
-        %h4
-          Reviews
-        = render AddReviewDropdownComponent.new(bs_request: bs_request, user: User.session,
-                                                can_add_reviews: can_add_reviews,
-                                                my_open_reviews: my_open_reviews)
+        %h4.mt-2
+          Reviewers
+        = render partial: 'webui/request/beta_show_tabs/ask_for_review', locals: { can_add_reviews: can_add_reviews }
```

Comment:
Why do you feel it is _counter intuitive_?

Let me drive you through the rationale I went into: we have **two** action buttons here, one to add a new reviewer, one to submit a review. From the button action type perspective, the submit one is stronger, and it has to go at the bottom. Therefore the add one becomes an additional action of the box, and it is very related to the title of it: in the end it adds a reviewer. It can go to the right top because it belongs more to some potential _toolbar_ with properties and actions within that context, and toolbars always (almost) go at the top.

Anyway your early feedback is important, I'll keep thinking about it. Thank you.

Reply:
After thinking it twice and trying to _merge_ all our points of view, I came up with the following:
- the **Review** button is related to the reviewer list but it is moreover related to the Request itself, so that action button belongs more to the whole page (thinking about GitHub PRs, to name one). Therefore I will move the **Review** button outside of the reviewer side box and place it on top of the conversation tab content.
- because of the change as described above, the **ask for a review** button can gain back its original place at the bottom of the reviewer list, because now it is the only button related to that box and on top of that it can gain back a default action button style
![refactoring-alt](https://user-images.githubusercontent.com/7080830/196959850-93c387c1-8e21-4e74-9cdb-6de183fc729b.png)

Thanks for the discussion we came up with, I think, and even better solution of all the other options.

Reply:
Resolving this conversation as we agreed after offline discussion.